tests/cluster_sanity/test_cluster_alerts.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import logging
+from tests.cluster_sanity.utils import verify_no_listed_alerts_on_cluster, verify_prometheus_connection
+
+TIMEOUT_3MIN = 3*60
```

Comment:
maybe it will be worth to create a constants file and re-use variables like this

Reply:
I'll create it , right now there were not many variables I added it here